src/Provider/Cache/ProviderCache.php

```diff
@@ -40,16 +40,22 @@ class ProviderCache implements Provider
      */
     protected $lifetime;
 
+    /**
+     * If true, include the real provider name into the cache key
+     */
+    protected bool $separateCache;
```

Comment:
```suggestion
    private bool $separateCache;
```

Reply:
Ok, I just wanted to keep consistency with other properties, but TBH I don't have the need so let's update it

Reply:
It's done :)